test/acceptance/lettuce/integration/terrain.py

```diff
@@ -20,16 +20,16 @@
 # For those usages not covered by this license please contact with
 # iot_support at tid dot es
 """
-
 __author__ = 'Jon Calderin Goñi (jon.caldering@gmail.com)'
 
+
 from lettuce import before, world, after
 from integration.tools.general_utils import stop_mock, drop_all_test_databases, check_properties, get_cb_pid, stop_cb
 from iotqautils.iotqaLogger import get_logger
 
 @before.all
 def before_all():
-    world.log = get_logger('lettuce', world.config['environment']['log_level'], True, True, 'logs/lettuce.log')
+    world.log = get_logger('lettuce', world.config['environment']['log_level'], True) #, True, 'logs/lettuce.log')
```

Comment:
Maybe a "Fixme: last parameters are commented by {some reason}" could be added above this line?